INFO:__main__:resolving package sklearn...
ERROR:resolvers:error: unable to find working package for {'package': 'scikit-learn', 'versions': ['0.21.3', '0.20.4', '0.19.2']}
INFO:__main__:resolving package pandas...
INFO:__main__:resolving package pandas succeeded
INFO:__main__:resolving package sklearn failed
ERROR:__main__:Exception occurred in outer context: 'NoneType' object has no attribute 'run_line_magic'
